homeassistant/components/qwikswitch.py

```diff
@@ -136,61 +134,69 @@ def setup(hass, config):
     url = config[DOMAIN][CONF_URL]
     dimmer_adjust = config[DOMAIN][CONF_DIMMER_ADJUST]
 
-    qsusb = QSUsb(url, _LOGGER, dimmer_adjust)
+    def callback_value_changed(qsdevices, key, new): \
+            # pylint: disable=unused-argument
+        """Update entiry values based on device change."""
+        entity = hass.data[DOMAIN].get(key)
```

Comment:
The component should not know about the entity instances. That's a concern of the platforms. You should probably refactor this.

Reply:
You can eg use our dispatch helper to signal the entities to update state.